tf-modules/cumulus/ecs_cluster_instance_autoscaling_cf_template.yml.tmpl

```diff
@@ -39,6 +39,11 @@ Resources:
           sed -i '/^\s*DOCKER_STORAGE_OPTIONS=/d' /etc/sysconfig/docker-storage
           echo 'DOCKER_STORAGE_OPTIONS="--storage-driver ${docker_storage_driver}"' >> /etc/sysconfig/docker-storage
 %{ endif ~}
+
+%{ if include_docker_cleanup_cronjob == true ~}
+          echo '* * * * * sudo sh -c "docker ps -q | xargs docker inspect --format='\{{.State.Pid}}' | xargs -IZ fstrim /proc/Z/root/"' | crontab -
```

Comment:
Given this is the root user, sudo probably isn't required here.    That said, we can fix it in main.

Reply:
Also per-minute seems like it might have adverse effects in certain situations.    Approving pending a fix on the main PR.

Reply:
I checked with Nga and they did still see issues with a 45-minute cadence at high load. For the time being a minute would be their preference since it is what they know works.